### PR TITLE
feat: add native update step

### PR DESCRIPTION
**Pull request by justin-betty**
add native update step

### CODE REVIEW

File: functions/update/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { parseAssignedProperties, fetchRecord } from '../utils';
+
+const update = async ({
+  selectedRecord: {
+    data: { id: recordId },
```
Comment (thomasvanophem) [inline functions/update/index.js:5]: Maybe we could rename `recordId` to `id` as we no longer destructure the data returned from the DataAPI to get the record id.

File: __tests__/support/gql/index.js [hunk 2]
```diff
@@ -88,6 +89,10 @@ const root = {
       id,
     };
   },
+  updateUser({ id, input }) {
+    const user = new User(id, userDatabase[1]);
```
Comment (thomasvanophem) [inline __tests__/support/gql/index.js:93]: I think we should change the `userDatabase` to only contain `User` instances. Something like 
```
const userDatabase = {
  1: new User(1, {
    id: 1,
    firstName: 'John',
    lastName: 'Doe',
    age: 30,
    username: 'test@test.test',
    password: 'test1234',
  }),
};
```

Then we don't need to use `new User()` to get the instance.
Think this `userDatabase[1]` should be `userDatabase[id]`,

File: functions/update/index.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    model: { name: modelName },
+  },
+  mapping,
+}) => {
+  const assignProperties = parseAssignedProperties(mapping);
+
+  const input = {
+    ...assignProperties,
```
Comment (thomasvanophem) [inline functions/update/index.js:13]: Is the `updatedAt` set by the DataAPI?

**Review by thomasvanophem — CHANGES_REQUESTED** [review]

File: functions/update/index.js (hunk 3 above)
Comment (justin-betty) [reply functions/update/index.js:13]: yes it is. I validated that by removing it, publishing it and run it

Comment (thomasvanophem) [reply functions/update/index.js:13]: Great! Can you check if `createdAt` and `updatedAt` are also set by the DataAPI in the `create` mutation? And if so, remove those fields?

Comment (justin-betty) [reply functions/update/index.js:13]: Removed as well

**Review by thomasvanophem — APPROVED** [review]

**Pull request merged**